bot/modules/orders/messages.js

```diff
@@ -21,17 +21,20 @@ exports.listOrdersResponse = async (orders, i18n) => {
 
     if (typeof order.amount !== 'undefined') amount = order.amount;
     const timeToExpire = getTimeToExpirationOrder(order, i18n);
-    return [
+    let details = [
```

Comment:
almost done, for some weird reason for javascript a constant can be modified if it is an array, prettier complains to use `const` instead of `let` for details

Reply:
should be fixed now